karma.conf.js

```diff
@@ -1,15 +1,18 @@
-import paths from './paths';
-import statConfig from './webpack-stats';
-import friendlyFormatter from 'eslint-friendly-formatter';
-import exitPlugin from './webpack-exit-plugin';
-import ENV from './ENV';
+/* eslint-disable no-var */
+require('babel-core/register');
```

Comment:
Global `babel-node` was gagging.  After several attempts at fixing it, decided to just re-write the karma.conf in ES5, this was the only file using `babel-node`.  All the files this one requires are compiled with babel via the magical `register` hook we see here.

>npm test executes karma, see package.json.